OpenEdXMobile/res/layout-land/empty_screen_layout.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/empty_screen_layout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="@dimen/edx_default_margin"
+    android:background="@drawable/white_rounded_borders"
+    android:elevation="@dimen/edx_quarter_margin"
+    android:fillViewport="true">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <FrameLayout
+            android:id="@+id/content_error_icon"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:background="@color/neutralXLight"
```

Comment:
we need to create the background with round corners for `neutralXLight`, cuz for the landscape didn't shows the round corners.